docs/develop/python/message-passing.mdx

```diff
@@ -551,23 +602,28 @@ from temporalio.common import RawValue
         ...
 ```
 
+This sample creates a `dynamic_signal` Signal.
+When an unregistered or unrecognized Signal arrives with a `(self, str, Sequence[RawValue])` signature, dynamic assignment will use this handler to manage the Signal.
+It is responsible for transforming the sequence contents into usable data in a form that the method's logic can process and act on.
+
 ### Set a Dynamic Workflow {#set-a-dynamic-workflow}
 
```

Comment:
This section needs more work and isn't ready to go out yet. One option would be to leave the existing content unchanged and deal with it in subsequent work, after Update launch. It is in the wrong place though (dynamic workflows and activities are not related to message-passing).